components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -115,6 +115,12 @@ export class Panel extends React.Component<Props, State> {
     this.actions.getGrantCaptcha()
   }
 
+  onBackupWallet = () => {
+    chrome.tabs.create({
+      url: 'chrome://rewards#backup-restore'
+    })
```

Comment:
We should clear notification when user clicks backup now

Reply:
done